### PR TITLE
Admin UI to manage application users

### PR DESCRIPTION
**Pull request by bp323**
It would be useful to have a UI where global admins (and tenant admins) can manage application users.
Things like:
 - Promoting to tenant admin
 - Changing email address
 - Changing display name
etc..

screen shot 2015-04-16 at 17 05 24

### CODE REVIEW

**Comment by bp323** [comment]
Functionality available in the global admin interface. Just need to follow up on the tests.

**Comment by bp323** [comment]
Reassigning to @Coenego for review.

File: apps/admin/ui/js/index.js [hunk 1]
```diff
@@ -512,6 +546,117 @@ define(['gh.core', 'gh.constants', 'chosen', 'validator'], function(gh, constant
...
+     * @private
+     */
+    var getUsers = function() {
+        // Get the ID of the app to search for users in
+        var appId = parseInt($(this).closest('.panel-group').data('appid'), 10);
+        // Get the search query
+        var query = $($(this).closest('.panel-group').find('.gh-striped-container-search input')).val();
+
+        gh.api.userAPI.getUsers(appId, null, null, query, function(err, data) {
+            renderAppUsersResults(appId, data);
```
Comment (Coenego) [inline apps/admin/ui/js/index.js]: Maybe we should add an error handler here?

File: tests/qunit/tests/js/api.user.js [hunk 2]
```diff
@@ -737,15 +737,18 @@ require(['gh.core', 'gh.api.tests'], function(gh, testAPI) {
...
                     // Verify that the admin status can be updated without errors
                     gh.api.userAPI.updateAdminStatus(user.id, true, function(err, data) {
-
-                        /**
-                         * TODO: wait for back-end implementation
-                         *
                         assert.ok(!err, 'Verify that the admin status can be updated without errors');
-                        assert.ok(data, 'Verify that the updated user is returned');
-                         */
+                        assert.strictEqual(data.isAdmin, true, 'Verify that the admin status can be updated without errors');
```
Comment (Coenego) [inline tests/qunit/tests/js/api.user.js]: Maybe we should also add `ok(!err, '...')` here?

Comment (bp323) [inline tests/qunit/tests/js/api.user.js]: It's on the line above I think?

Comment (Coenego) [inline tests/qunit/tests/js/api.user.js]: Oops :) Totally didn't notice that one

**Comment by Coenego** [comment]
I'm currently seeing the following error when opening the configurations for an app: `Uncaught TypeError: Cannot read property 'results' of undefinedrenderAppUsersResults @ index.js:64`

**Comment by Coenego** [comment]
Back to @bp323 for follow-up

**Comment by bp323** [comment]
Over to @Coenego for review

**Pull request merged**